### ISSUE TITLE
atari 5200 default controller modes (mouse)

### ISSUE THREAD

**Issue by goofyseeker311** [MiSTer-devel/Atari7800_MiSTer#28]
for joust, the default atari 5200 controller mode, during first start, is mouse, instead of keyboard.
even if you specify the keyboard controls, and/or remove mouse. keyboard starts working only after rebooting.

edit: mouse cant be connected at all to the mister, if you want keyboard control, for atari5200.

**Comment by Kitrinx** [comment]
Im sorry, is this for atari 5200? That is not part of this core.

**Comment by goofyseeker311** [comment]
well, there is no repo for the 5200 specifically.

I'll post this to the atari800 (5200) core issues.

**Comment by Kitrinx** [comment]
Yes, they are the same core (800 and 5200) it's located here, thank you https://github.com/MiSTer-devel/Atari800_MiSTer
